handlers/server_spec_handler.go

```diff
@@ -40,13 +39,13 @@ func (h *Handler) ServerVersionHandler(w http.ResponseWriter, r *http.Request) {
 	// }
 
 	// compare the server build with the target build
-	res, err := CheckLatestVersion(version.Build)
+	isOutdated, latestVersion, err := CheckLatestVersion(version.Build)
 	if err != nil {
 		h.log.Error(err)
 	} else {
 		// Add "Latest" and "Outdated" fields to the response
-		version.Latest = res.Current
-		version.Outdated = &res.Outdated
+		version.Latest = latestVersion
+		version.Outdated = isOutdated
```

Comment:
@piyushsingariya will `isOutdated` still be true even if the channel is `edge`? :thinking:

Reply:
isOutdated will be false because edge will be always same as the latest release tag.